packages/nodes-base/nodes/ExecuteWorkflow/ExecuteWorkflow/ExecuteWorkflow.node.ts

```diff
@@ -14,9 +14,30 @@ import { loadWorkflowInputMappings } from './methods/resourceMapping';
 import { generatePairedItemData } from '../../../utils/utilities';
 import { getWorkflowInputData } from '../GenericFunctions';
 
+function getWorkflowInputValues(this: IExecuteFunctions) {
```

Comment:
Maybe using `map` instead of pushing to array in a loop would be more performant for larger inputs:
```suggestion
function getWorkflowInputValues(this: IExecuteFunctions) {
	const inputData = this.getInputData();

	return inputData.map((item, itemIndex) => {
		const itemFieldValues = this.getNodeParameter(
			'workflowInputs.value',
			itemIndex,
			{},
		) as IDataObject;

		return {
			json: {
				...item.json,
				...itemFieldValues,
			},
			index: itemIndex,
			pairedItem: {
				item: itemIndex,
			},
		};
	});
}
```